topics/ecology/tutorials/de-novo-rad-seq/tutorial.md

```diff
@@ -20,7 +20,10 @@ tags:
   - EBV dataset
   - EBV workflow
 contributors:
```

Comment:
Splitting contributions like this needs `contributions` (not `contributors`) as the top-level attribute.